packages/lesswrong/lib/voting/votingSystems.tsx

```diff
@@ -346,7 +346,9 @@ registerVotingSystem<{preVote: boolean}, {preVoteCount: number}>({
   isAllowedExtendedVote: (user) => {
     try {
       assertUserCanVoteInDonationElection(user);
-      return {allowed: true};
+      // TODO: uncomment this if we want to run another election
+      // return { allowed: true };
+      return { allowed: false, reason: 'Pre-voting has closed. Click "Vote in the Election" above if you would like to cast your real vote' };
```

Comment:
This won't actually be returned because voting is blocked before actually submitting